components/src/lib/components/Modal/ModalContent.tsx

```diff
@@ -0,0 +1,69 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+
+interface ModalContentProps {
+  children: ReactElement[] | ReactElement;
+  position: 'top' | 'bottom' | 'center';
+  size?: 'small' | 'medium' | 'large';
+  styleProps?: React.CSSProperties;
+  ref?: React.RefObject<HTMLDivElement | null>;
+}
+
+type SizeKey = 'small' | 'medium' | 'large';
+
+const pxBySize: Record<SizeKey, number> = {
+  small: 320,
+  medium: 400,
+  large: 600,
+};
+
+const radius = {
+  top: '0px 0px 8px 8px',
+  center: '8px',
+  bottom: '8px 8px 0px 0px',
+};
+
+function ModalContent({
+  children,
+  position,
+  size,
+  styleProps,
+  ref,
+}: ModalContentProps) {
+  return (
+    <StyledModal
+      ref={ref}
+      id="modal-content"
+      position={position}
+      size={size}
+      role="dialog"
+      aria-modal="true"
+      styleProps={styleProps}
+    >
```

Comment:
props를 그대로 넘겨줘도 좋았을 것 같아요 ㅎㅎ 어차피 가공을 안 하고 있어서...
```suggestion
  props: ModalContentProps) {
  return (
    <StyledModal
      id="modal-content"
      role="dialog"
      aria-modal="true"
      {...props}
    >
```
요로코롬..